examples/apexlog/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
```

Comment:
This comment doesn't look useful because it doesn't tell anything new.

Reply:
Removed it.

Reply:
This is still present.